docs/src/components/types/prop.tsx

```diff
@@ -29,13 +29,13 @@ export const Prop: React.FC<PropProps> = props => {
         </Table.Cell>
 
         <Table.Cell>
-          <Text.Code>{props.type}</Text.Code>
```

Comment:
Why the switch here?

Reply:
The styling applied by `Text.Code` looks clunky and this looks closer to the current docs prop table.
![image](https://user-images.githubusercontent.com/48542/94285608-1fb98400-ff11-11ea-8644-b7240d8660e9.png)